src/c99-flex.skl

```diff
@@ -52,7 +52,12 @@ m4_define([[M4_HOOK_STATE_ENTRY_FORMAT]], [[	&yy_transition[$1],
 ]])
 m4_define([[M4_HOOK_NORMAL_STATE_CASE_ARM]], [[	case $1:]])
 m4_define([[M4_HOOK_EOF_STATE_CASE_ARM]], [[	case YY_STATE_EOF($1):]])
-m4_define([[M4_HOOK_EOF_STATE_CASE_FALLTHROUGH]], [[		/* FALLTHROUGH */]])
+m4_define([[M4_HOOK_EOF_STATE_CASE_FALLTHROUGH]], [[
+	#ifdef __clang__
+	__attribute__((fallthrough));
```

Comment:
NOT my project, but I'd probably suggest instead of doing `#ifdef clang` to use some sort of `has_cpp_attribute`/`has_attribute` series of checks to try to do `[[fallthrough]]` if supported, else `__attribute__((fallthrough)`` if that is supported.